data/lm/generate_package.py

```diff
@@ -124,24 +94,18 @@ def main():
     )
     parser.add_argument(
         "--force_utf8",
+        type=str,
```

Comment:
Giving the parameter a string type should avoid the implicit casting.

Reply:
This isn't what I meant. Parameters are string typed by default anyway, this doesn't change anything. What I mean is doing e.g. `if foo:` when `foo` is a Tristate, which can lead to bugs since the distinction between None and False is not handled.

Reply:
Keep the Tristate class.

Reply:
But i did move the parameter check, to avoid the None and False distinction.

Reply:
The None and False confusion can happen anywhere in the code where you can implicitly convert a tristate value to a boolean. This is why the Tristate class was introduced. It's not about validating input from command line arguments. Please keep the Tristate class.

Reply:
Do you use the Tristate class somewhere else?
Else i don't really understand why we still need it here. I only have comparisons with True/False or Strings left.

Reply:
I've already explained, twice...